app/controllers/api/databases.php

```diff
@@ -3532,8 +3532,10 @@ function updateAttribute(
 
         $processDocument($collection, $document);
 
-        $queueForDeletes
-            ->setType(DELETE_TYPE_AUDIT)
+        $queueForDatabase
+            ->setType(DATABASE_TYPE_DELETE_DOCUMENT)
+            ->setDatabase($database)
+            ->setCollection($collection)
```

Comment:
This is the delete document endpoint.. Why do we need to send this to a worker? We delete the document in the API itself right?

Reply:
I think this is for delete the document audit and not the document itself. We can leave it as is.